gui/src/app/pages/HomePage/LoadProjectWindow.tsx

```diff
@@ -103,9 +106,37 @@ const LoadProjectWindow: FunctionComponent<LoadProjectWindowProps> = ({
     }
   }, [filesUploaded, importUploadedFiles]);
 
+  const [browserProjectTitles, setBrowserProjectTitles] = useState<string[]>(
+    [],
+  );
+  useEffect(() => {
+    const bpi = new BrowserProjectsInterface();
+    bpi.listProjects().then((titles) => {
+      setBrowserProjectTitles(titles);
+    });
+  }, []);
```

Comment:
Note that there is a race condition here if you have two tabs open: saving (or deleting) a project in one will not update the other.

I think this is probably okay, it's not really an intended operational mode or likely to come up, but I wanted to point it out.

Reply:
I see what you mean, but this is not going to be a problem since retrieving from browser storage is fast enough.

Reply:
I don't think it's a matter of save/retrieve speed--it's if you have the window open already, there isn't anything triggering an update. (At least that's how it looked when I was playing around with it.)

Again, I think the steps to trigger are kind of far-fetched, so I'm not worried, but just want to note everything I see.

Reply:
Oh I see! Didn't read carefully enough :)